src/libraries/Microsoft.PowerFx.Core/Public/Values/BooleanValue.cs

```diff
@@ -9,6 +10,8 @@ namespace Microsoft.PowerFx.Types
 {
     public class BooleanValue : PrimitiveValue<bool>
     {
+        internal static readonly IReadOnlyList<FormulaType> AllowedListConvertToBoolean = new FormulaType[] { FormulaType.String, FormulaType.Number, FormulaType.Boolean };
```

Comment:
>AllowedListConvertToBoolean [](http://example.com/codeflow?start=60&length=27)

For these Allowed list, can we:
1. add a comment about which library function they correspond to 
2. add an enforcement check in Library.  So that if we did miss something on this list, we'll get a failure from the .txt

Reply:
updated this. thanks